source/cli.js

```diff
@@ -91,11 +91,22 @@ updateNotifier({pkg: cli.pkg}).notify();
 		...cli.flags
 	};
 
-	const isAvailable = flags.publish ? await isPackageNameAvailable(pkg) : false;
+	const {isAvailable = false, isUnknown = false} = await (async () => {
```

Comment:
This seems a bit counter-intuitive and overcomplicated to me. I'd create another method for this (i.e. `isPackageAvailable`) and return a proper dictionary containing both `isAvailable` and `isUnknown`.

Reply:
On second thought, an even simpler solution would be to create a sort-of enum for the return value. This would enable us to only return a single value, which would simplify usage later on.

Reply:
Amended PR per feedback.